pkg/alerts/location_failure_conditions.go

```diff
@@ -0,0 +1,96 @@
+package alerts
+
+import (
+	"fmt"
+)
+
+// ListLocationFailureConditions returns alert conditions for a specified policy.
+func (alerts *Alerts) ListLocationFailureConditions(policyID int) ([]*LocationFailureCondition, error) {
+	response := locationFailureConditionListResponse{}
+	locationFailureConditions := []*LocationFailureCondition{}
+	queryParams := listLocationFailureConditionsParams{
+		PolicyID: policyID,
+	}
+
+	nextURL := fmt.Sprintf("/alerts_location_failure_conditions/policies/%d.json", policyID)
+
+	for nextURL != "" {
+		resp, err := alerts.client.Get(nextURL, &queryParams, &response)
+
+		if err != nil {
+			return nil, err
+		}
+
+		locationFailureConditions = append(locationFailureConditions, response.LocationFailureConditions...)
+
+		paging := alerts.pager.Parse(resp)
+		nextURL = paging.Next
+	}
+
+	return locationFailureConditions, nil
+}
+
+// CreateLocationFailureCondition creates an alert condition for a specified policy.
+func (alerts *Alerts) CreateLocationFailureCondition(condition LocationFailureCondition, policyID int) (*LocationFailureCondition, error) {
+	reqBody := locationFailureConditionRequestBody{
+		LocationFailureCondition: condition,
+	}
+	resp := locationFailureConditionCreateResponse{}
+
+	u := fmt.Sprintf("/alerts_location_failure_conditions/policies/%d.json", policyID)
+	_, err := alerts.client.Post(u, nil, &reqBody, &resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return &resp.LocationFailureCondition, nil
+}
+
+// UpdateLocationFailureCondition updates an alert condition.
+func (alerts *Alerts) UpdateLocationFailureCondition(condition LocationFailureCondition) (*LocationFailureCondition, error) {
+	reqBody := locationFailureConditionRequestBody{
+		LocationFailureCondition: condition,
+	}
+	resp := locationFailureConditionCreateResponse{}
+
+	u := fmt.Sprintf("/alerts_location_failure_conditions/%d.json", condition.ID)
+	_, err := alerts.client.Put(u, nil, &reqBody, &resp)
+
+	if err != nil {
+		return nil, err
+	}
+
+	// resp.Condition.PolicyID = condition.PolicyID
```

Comment:
@zlesnr I think we've gone back and forth on this a couple times.  I don't think I see a problem with including the policy ID in the condition definition, though I'm pretty sure we didn't do it for the other condition types.  If you want to add it, go ahead but let's enter an issue to make it consistent in other condition types as well.

Reply:
I mistakenly left that comment in there from the resource that I copied.  Commented here because the object itself doesn't have a PolicyID, and I didn't go digging to see if the source object I copied did or not.  Definitely consistency, though here I think modeling the API(s) make sense.  I can dig a little more on what the other resources are doing there.

Reply:
It looks like the PolicyID on an `alerts.Condition` also has a PolicyID attached to the object that is not returned from the API.

https://rpm.newrelic.com/api/explore/alerts_conditions/list

I'm slightly of the opinion that we shouldn't add extra fields to objects, but maybe there is a good use case to keep the policy ID attached to the condition.  I'll file a follow up issue to get consistency.

Reply:
https://github.com/newrelic/newrelic-client-go/issues/120 to follow up.